Resources/Maps/_NF/Shuttles/loader.yml

```
@@ -740,6 +740,14 @@ entities:
     - type: Transform
       pos: 1.5,6.5
       parent: 1
+- proto: ComputerWallmountBankATM
```

Withdraw-only, if at all.
```suggestion
- proto: ComputerWallmountWithdrawBankATM
```